### PR TITLE
WinUI-2902 : Prepare getting started md file in the rating control UG.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Prepared getting started content for the rating control UG.

### CODE REVIEW

File: winui/Rating/Getting-Started.md [hunk 1]
```diff
@@ -10,3 +10,196 @@ documentation: ug
 # Getting Started with WinUI Rating
 
 This section explains the steps required to add the Rating control and covers only the basic features needed to get started with Syncfusion SfRating control.
+
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Getting-Started.md:13]: Changed the description as mentioned above.

File: winui/Rating/Getting-Started.md [hunk 2]
```diff
@@ -10,3 +10,196 @@ documentation: ug
...
+## Creating an application with WinUI Rating control
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Getting-Started.md:14]: Changed the content as mentioned above.

File: winui/Rating/Getting-Started.md [hunk 3]
```diff
@@ -10,3 +10,196 @@ documentation: ug
...
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/get-started-winui3-for-desktop).
+2. Add reference to NuGet.
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Getting-Started.md:17]: Added the reference.

File: winui/Rating/Getting-Started.md [hunk 4]
```diff
@@ -10,3 +10,196 @@ documentation: ug
...
+rating.Items.Add(new SfRatingItem());
+rating.Items.Add(new SfRatingItem());
+rating.Value = 2;
+           
+{% endhighlight %}
+{% endtabs %}
+
+[!Rating control default in WinUI](Rating_images/winui_rating_default.png)
+
+## Rating control using TemplateSelector Property with Images
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Getting-Started.md:67]: Removed the template selector.

File: winui/Rating/Getting-Started.md [hunk 5]
```diff
@@ -10,3 +10,196 @@ documentation: ug
...
+    xmlns:local="using:GettingStarted"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:syncfusion="using:Syncfusion.UI.Xaml.Editors"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Grid>
+     <syncfusion:SfRating 
+          Value="2">
+         <syncfusion:SfRating.Items>
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Getting-Started.md:37]: Changed the code snippet with itemscount.

File: winui/Rating/Getting-Started.md [hunk 6]
```diff
@@ -10,3 +10,196 @@ documentation: ug
...
 This section explains the steps required to add the Rating control and covers only the basic features needed to get started with Syncfusion SfRating control.
+
+## Creating an application with WinUI Rating control
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/get-started-winui3-for-desktop).
+2. Add reference to NuGet. 
+3. Import the control namespace `Syncfusion.UI.Xaml.Editors` in XAML or C# code.
+4. Initialize the SfRating control.
+
+{% tabs %}
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Getting-Started.md:21]: used two heading for items and itemscount.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Rating/Getting-Started.md [hunk 7]
```diff
@@ -1,12 +1,89 @@
 ---
 layout: post
-title: Getting Started with WinUI Rating control | Syncfusion
-description: Learn all about getting started with the Syncfusion WinUI Rating(SfRating) control, its elements, and more here.
+title: Getting started with WinUI Rating control | Syncfusion
+description: Learn here all about getting started with the Syncfusion WinUI Rating(SfRating) control and its basic features.
```
Comment (OnyimbiNelson) [inline winui/Rating/Getting-Started.md:4]: description: Learn all about getting started with the Syncfusion WinUI Rating(SfRating) control and its basic features here.

Comment (Priyanka-Vijayasankar) [reply winui/Rating/Getting-Started.md:4]: Changed the content description as mentioned above.

**Pull request merged**